sdk/include/BooleanQuery.h

```diff
@@ -9,6 +9,11 @@
 // is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
 // or implied. See the License for the specific language governing permissions and limitations under the License.
 
+
+#ifndef MILVUS_SDK_INCLUDE_BOLLEANQUERY_H_
+#define MILVUS_SDK_INCLUDE_BOLLEANQUERY_H_
+
+
 #pragma once
```

Comment:
"#pragma once" has the same effect, while it is not necessary.

Reply:
thanks~

Reply:
But don't you think `#ifndef` is a more safety approach since it also can prevent same codes in different header files included by twice : ) ?